src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/default-transcripts-list-item/DefaultTranscriptListItem.test.tsx

```diff
@@ -88,10 +88,10 @@ describe('<DefaultTranscriptListItem />', () => {
     expect(queryByTestId('transcriptsListItemInfo')).toBeTruthy();
   });
 
-  it('displays selected transcript', () => {
-    const { container } = renderComponent({ isDefault: true });
+  it('displays Ensembl canonical transcript with the correct label', () => {
```

Comment:
The test isn't really checking what it claims to be checking. The description of the test talks about Ensembl canonical transcript, but the code on line 95 is checking the mane metadata (which means that this also happens to be a MANE select transcript)?

If you really want to test the content of the label, you should probably test the `TranscriptQualityLabel` component, move this test over there, and add some other test cases:

- canonical + mane select => uses transcript.metadata.mane.label
- canonical without mane select => uses transcript.metadata.canonical.label
- non-canonical, with mane metadata => uses transcript.metadata.mane.label
- non-canonical, no mane metadata => returns null